modules/exploits/windows/local/bypassuac_dotnet_profiler.rb

```diff
@@ -63,7 +63,6 @@ def initialize(info = {})
   end
 
   def check
-    get_version_info
```

Comment:
I think this is just dead code, as `get_version_info` is in `is_uac_enabled?` already

Reply:
This is all new to me. Am I to replace something for this? If so, where?